wework/src/styles/globals.css

```
@@ -1332,6 +1332,27 @@ button:disabled {
+      transform: rotate(44deg) scale(0.96);
+    }
+    42% {
+      transform: rotate(108deg) scale(0.98);
+    }
+    64% {
+      transform: rotate(158deg) scale(1.06);
+    }
+    84% {
+      transform: rotate(176deg) scale(1.02);
+    }
+    to {
```

Stylelint (17.14.0)

[error] Expected "to" to be "100%" (keyframe-selector-notation)

(keyframe-selector-notation)